EventFilter/CTPPSRawToDigi/python/totemTimingRawToDigi_cfi.py

```diff
@@ -1,13 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+
 from EventFilter.CTPPSRawToDigi.totemVFATRawToDigi_cfi import totemVFATRawToDigi
 
 totemTimingRawToDigi = totemVFATRawToDigi.clone(
     subSystem = cms.string('TotemTiming'),
```

Comment:
```suggestion
    subSystem = 'TotemTiming',
```
... and the same in the following